javascript/exercises/creditcard.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Exercises: Animation</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link href="../../common/css/bootstrap.css" rel="stylesheet">
+  </head>
+ <body>
+<div class="container">
+
+<h1>Homework: Credit Card Validation</h1>
+
+<p>You're starting your own credit card business. You've come up with a new way to validate credit cards with a simple function called <code>validateCreditCard</code> that returns <code>true</code> or <code>false</code>.</p>
+
+<p>Here are the rules for a valid number:
+<ul>
+	<li>Number must be 16 digits, all of them must be numbers</li>
+	<li>You must have at least two different numbers represented (all of the numbers cannot be the same)</li>
+	<li>The final number must be even</li>
+	<li>The sum of all the digits must be greater than 16</li>
+</ul>
+</p>
+
+<p>The following credit card numbers are valid:</p>
+<pre><code>9999-9999-8888-0000
+6666-6666-6666-1666
+</code></pre>
+
+<p>The following credit card numbers are invalid:
+  <ul>
+    <li><code>a923-3211-9c01-1112</code> <em>invalid characters</em></li>
+    <li><code>4444-4444-4444-4444</code> <em>only one type of number</em></li>
+    <li><code>1211-1111-1111-1112</code> <em>sum less than 16</em></li>
```

Comment:
the sum here is 18. Either change the rule or change the example to 1111-1111-1111-1111.